openquake/hazardlib/tests/gsim/parker_2020_test.py

```diff
@@ -0,0 +1,288 @@
+# -*- coding: utf-8 -*-
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+#
+# Copyright (C) 2012-2020 GEM Foundation
+#
+# OpenQuake is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# OpenQuake is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with OpenQuake. If not, see <http://www.gnu.org/licenses/>.
+
+from openquake.hazardlib.gsim.parker_2020 import \
+    ParkerEtAl2020SInter, \
+    ParkerEtAl2020SInterAleutian, \
+    ParkerEtAl2020SInterAlaska, \
+    ParkerEtAl2020SInterCascadiaOut, \
+    ParkerEtAl2020SInterCascadiaSeattle, \
+    ParkerEtAl2020SInterCascadia, \
+    ParkerEtAl2020SInterCAMN, \
+    ParkerEtAl2020SInterCAMS, \
+    ParkerEtAl2020SInterJapanPac, \
+    ParkerEtAl2020SInterJapanPhi, \
+    ParkerEtAl2020SInterSAN, \
+    ParkerEtAl2020SInterSAS, \
+    ParkerEtAl2020SInterTaiwanE, \
+    ParkerEtAl2020SInterTaiwanW, \
+    ParkerEtAl2020SSlab, \
+    ParkerEtAl2020SSlabAleutian, \
+    ParkerEtAl2020SSlabAlaska, \
+    ParkerEtAl2020SSlabCascadiaOut, \
+    ParkerEtAl2020SSlabCascadiaSeattle, \
+    ParkerEtAl2020SSlabCascadia, \
+    ParkerEtAl2020SSlabCAMN, \
+    ParkerEtAl2020SSlabCAMS, \
+    ParkerEtAl2020SSlabJapanPac, \
+    ParkerEtAl2020SSlabJapanPhi, \
+    ParkerEtAl2020SSlabSAN, \
+    ParkerEtAl2020SSlabSAS, \
+    ParkerEtAl2020SSlabTaiwanE, \
+    ParkerEtAl2020SSlabTaiwanW
+
```

Comment:
You do not need to import lots of classes and write lots of TestCase subclasses: since you are using parametric GMPEs you can use the approach followed for instance in https://github.com/gem/oq-engine/blob/master/openquake/hazardlib/tests/gsim/nshmp_2014_test.py#L79:
a single test class with a test method for each combination of the parameters.